data/cache/.gitignore

```diff
@@ -1,2 +0,0 @@
-*
```

Comment:
This file appears to have been deleted by accident.

Reply:
I see that you resolved this comment, but the file is still marked as deleted. Could you try again to restore it?

Reply:
I went ahead and fixed this while I was working in the PR anyway.